SU-SWS/ace-sdssgryphon
# SDSS-225: color fixes to news and events lists on green row variant

# READY FOR REVIEW

# Summary
- updating colors for a11y on news/events on green row variant

# Review By (Date)
- soon

# Urgency
- Normal 

# Steps to Test

1. Pull in this change.
2. Clear caches.
3. look at this page and check the news and events on green.
4.  a11y-qa-list-and-teaser-display

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
https://stanfordits.atlassian.net/browse/SDSS-225

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
